OData/src/System.Web.OData/OData/Formatter/Serialization/ODataCollectionSerializer.cs

```diff
@@ -52,13 +53,17 @@ public ODataCollectionSerializer(ODataSerializerProvider serializerProvider)
         public sealed override ODataValue CreateODataValue(object graph, IEdmTypeReference expectedType,
             ODataSerializerContext writeContext)
         {
-            if (graph == null)
+            var config = writeContext.Request.GetConfiguration();
```

Comment:
But what exception would you like me to throw?  What message?  How do I expose this "impossible" situation to the end-user (who should *never* see it)?  NullArgumentException("Request is null, this should only happen in unit tests that do not moq the request, end users should not see it") for example?